docs/user/README.md

```diff
@@ -0,0 +1,13 @@
+# Docker Registry Module
+
+## What is Docker Registry?
+
+<!-- TODO: add more information -->
```

Comment:
Docker registry module is a kubernetes operator that adds docker registry capability into a kubernetes cluster. It installs [distribution](https://distribution.github.io/distribution/) image; packaged and configured so that it is easily usable in kyma runtime.